leap_c/examples/hvac/env.py

```diff
@@ -26,13 +26,6 @@
     BestestParameters,
 )
 
-# Constants
-DAYLIGHT_START_HOUR = 6
-DAYLIGHT_END_HOUR = 18
-MEAN_AMBIENT_TEMPERATURE = convert_temperature(0, "celsius", "kelvin")
-MAGNITUDE_AMBIENT_TEMPERATURE = 5
-MAGNITUDE_SOLAR_RADIATION = 200
-
 
 class StochasticThreeStateRcEnv(MatplotlibRenderEnv):
     """Simulator for a three-state RC thermal model with exact discretization of Gaussian noise.
```

Comment:
This is by far the most complex environment, but it has the sparsest description (see other envs). This docstring should be comparable to the other envs. 
On top, we could also add a reference to the name of a paper in the docstring, if it makes sense and an appropriate paper exists.